gef-ui/server/controllers/about-exporter/index.js

```diff
@@ -16,18 +16,17 @@ const aboutExporter = async (req, res) => {
   const { status } = query;
 
   try {
-    const { exporterId } = await api.getApplication(applicationId);
-    const { details } = await api.getExporter(exporterId);
-    const industries = mappedIndustries(details.industries, JSON.stringify(details.selectedIndustry));
-    const isFinanceIncreasing = JSON.stringify(details.isFinanceIncreasing);
-    const probabilityOfDefault = JSON.stringify(details.probabilityOfDefault);
+    const { exporter } = await api.getApplication(applicationId);
+    const industries = mappedIndustries(exporter.industries, JSON.stringify(exporter.selectedIndustry));
+    const isFinanceIncreasing = JSON.stringify(exporter.isFinanceIncreasing);
+    const probabilityOfDefault = JSON.stringify(exporter.probabilityOfDefault);
 
     return res.render('partials/about-exporter.njk', {
-      smeType: details.smeType,
-      probabilityOfDefault: probabilityOfDefault !== 'null' ? probabilityOfDefault : null,
-      isFinanceIncreasing: isFinanceIncreasing !== 'null' ? isFinanceIncreasing : null,
-      selectedIndustry: details.selectedIndustry,
       applicationId,
+      smeType: exporter?.smeType,
+      probabilityOfDefault: probabilityOfDefault !== 'null' ? exporter.probabilityOfDefault : null,
+      isFinanceIncreasing: isFinanceIncreasing !== 'null' ? isFinanceIncreasing : null,
```

Comment:
I suppose same goes for `isFinanceIncreasing`

Reply:
Made loads simpler